core/domain/summary_services_test.py

```diff
@@ -362,3 +362,70 @@ def test_for_featured_explorations(self):
         }
         self.assertDictContainsSubset(
             expected_summary, featured_exploration_summaries[0])
+
+
+class TopRatedExplorationDisplayableSummariesTest(
+        test_utils.GenericTestBase):
+    """Test functions for getting displayable top rated exploration
+    summary dicts.
+    """
+
+    ALBERT_NAME = 'albert'
+    ALBERT_EMAIL = 'albert@example.com'
+
+    EXP_ID_1 = 'eid1'
+    EXP_ID_2 = 'eid2'
+
+    def setUp(self):
+        """Populate the database of explorations and their summaries.
+
+        The sequence of events is:
+        - (1) Albert creates EXP_ID_1.
+        - (2) Albert creates EXP_ID_2.
+        - (3) Albert publishes EXP_ID_1.
+        - (4) Albert publishes EXP_ID_2.
+        - (5) Admin user is set up.
+        """
+
+        super(TopRatedExplorationDisplayableSummariesTest, self).setUp()
+
+        self.admin_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+
+        self.save_new_valid_exploration(self.EXP_ID_1, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_2, self.albert_id)
+
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_1)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_2)
+
+        self.set_admins([self.ADMIN_USERNAME])
+
+    def test_for_top_rated_explorations(self):
+        """Note that EXP_ID_1 is public, and EXP_ID_2 is publicized.
+        The call to get_top_rated_explorations() should only return
+        [EXP_ID_2].
```

Comment:
Not sure I get this. If we are returning up to 6 top-rated explorations, shouldn't both be returned?

Also, you'll want some tests where the explorations actually have different ratings, although I agree it's also good to test the null case.